Plugins Contribution Guide (#9)
squaredup/squaredup-plugin-public: README.md

# SquaredUp Community Plugins Repository

> This repository contains third-party/community plugins for the SquaredUp Observability Portal product.

Reviewer: ```suggestion
This repository contains third-party/community plugins for [SquaredUp](https://squaredup.com).
```

## Introduction

> SquaredUp Cloud plugins are vital components of the SquaredUp Unified Observability Portal product. Each 

Reviewer: ```suggestion
Plugins are vital components of the SquaredUp product. Each 
```

## Introduction

> Plugins are vital components of the SquaredUp Unified Observability Portal product. Each 

Reviewer: ```suggestion
Plugins are vital components of SquaredUp. Each 
```

## Execution Environments

> A plugin may be executed in a cloud environment as part of the Observability Portal product or on a machine within a customer's premises (upon which the customer has installed a Relay Agent). If both environments are equally suitable for a plugin, it can

Reviewer: ```suggestion
A plugin may be executed within SquaredUp's SaaS environment or on a machine within a customer's premises (upon which the customer has installed a Relay Agent). If both environments are equally suitable for a plugin, it can
```

> be declared as "hybrid" (in which case, two separate plugins will appear in the Data Source Gallery, one with the label "On-Prem" added).

Reviewer: ```suggestion
be declared as "hybrid" (in which case, two separate plugins will appear in the Data Source list, one with the label "On-Prem" added).
```

> - `data_streams.json` - a file describing the streams of on demand data that the Observability Portal can request of the plugin.

Reviewer: ```suggestion
- `data_streams.json` - a file describing the streams of on demand data that SquaredUp can request of the plugin.
```

Can you remove any mentions of 'observability portal' etc.?
